core/src/server/handlers/http_handler_base.cpp

```diff
@@ -828,6 +836,12 @@ additionalProperties: false
         type: string
         description: overrides log level for this handle
         defaultDescription: <no override>
+    enable-tracing-headers:
```

Comment:
Please duplocate the description into https://github.com/userver-framework/userver/blob/115814a82a4a17cc4c83bd8eab8cf82e72c8d814/core/include/userver/server/handlers/http_handler_base.hpp#L48

Reply:
Good point, totally forgot about it